### PR TITLE
improved main and entrant pages and all links now opens in other window

### PR DESCRIPTION
**Pull request by Cub12**

### CODE REVIEW

File: frontend/src/app/components/entrant-page/entrant-page.component.sass
Comment (Mast0) [inline frontend/src/app/components/entrant-page/entrant-page.component.sass:1]: in my oppinion cummon line better then underscore

File: frontend/src/app/components/entrant-page/entrant-page.component.ts
Comment (Mast0) [inline frontend/src/app/components/entrant-page/entrant-page.component.ts:1]: why you delete this pice of code?

**Review by Mast0 — COMMENTED** [review]
I have only one qestion in my review

File: frontend/src/app/components/entrant-page/entrant-page.component.ts
Comment (Cub12) [reply frontend/src/app/components/entrant-page/entrant-page.component.ts:1]: cause it doesn't used anymore

Comment (Mast0) [reply frontend/src/app/components/entrant-page/entrant-page.component.ts:1]: ok

File: frontend/src/app/components/entrant-page/entrant-page.component.sass
Comment (Cub12) [reply frontend/src/app/components/entrant-page/entrant-page.component.sass:1]: okie, maybe, but there was an underline before that, so I didn't change it, we can think and change it later i guess

**Review by Mast0 — APPROVED** [review]
approved

**Pull request merged**